<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraServer</class>
 <widget class="QMainWindow" name="CameraServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraServer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
       background-color: rgb(230, 230, 230);
       color: black;                         
       font-weight: bold;              
    }
    QPushButton:disabled {                  
       background-color: rgb(211, 215, 207);
       color: gray;                 
       font-weight: normal;
    }</string>
        </property>
        <property name="text">
         <string>Start Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"> QPushButton {
       background-color: rgb(230, 230, 230);
       color: black;                         
       font-weight: bold;              
    }
    QPushButton:disabled {                  
       background-color: rgb(211, 215, 207);
       color: gray;                 
       font-weight: normal;
    }</string>
        </property>
        <property name="text">
         <string>Stop Server</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="avButton">
        <property name="styleSheet">
         <string notr="true">       background-color: rgb(230, 230, 230);
       color: black;                         
       font-weight: bold;    </string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="svButton">
        <property name="styleSheet">
         <string notr="true">       background-color: rgb(230, 230, 230);
       color: black;                         
       font-weight: bold;    </string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="viewfinderWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">minimumSize: 640x480</string>
      </property>
      <widget class="QLabel" name="previewLabel">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>781</width>
         <height>401</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
